pages/accountLists/[accountListId]/tools/appeals/index.page.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+import { loadSession } from 'pages/api/utils/pagePropsHelpers';
```

Comment:
We are missing 'Reason' on the excluded contacts. Do we want to add the headings for each column of information?
<img width="1313" alt="Screenshot 2024-08-21 at 12 42 59 PM" src="https://github.com/user-attachments/assets/d8df367c-eb9a-4021-9d9d-72b6a04723cc">
<img width="1322" alt="Screenshot 2024-08-21 at 12 43 16 PM" src="https://github.com/user-attachments/assets/ccbb0887-3635-4775-badd-f96c64e95285">

Reply:
Fixing this in a different PR - Reason is not on the GraphQL production